insights/parsers/ethtool.py

```diff
@@ -501,9 +501,10 @@ def parse_content(self, content):
         for line in content[2:]:
             if line.strip():
                 (key, value) = [s.strip() for s in line.split(":", 1)]
-                value = int(value)
-                self.data[key] = value
-                setattr(self, key.replace("-", "_"), value)
+                if 'n/a' not in value:
```

Comment:
Since there is not description I am assuming that handling `n/a` is causing the error. Can you please add some test to cover that changes?

Reply:
Yeap, @chenlizhong could you please add a relevant description for the issue and tests of the update?

Reply:
BTW, please check these checkboxes if already match. And , we have agreed to use the [Coventional commit messages](https://www.conventionalcommits.org/en/v1.0.0/) for the insights-core to make it a better public and open-sourced project. Here is a cheat sheet for your reference as well: https://cheatography.com/albelop/cheat-sheets/conventional-commits/

Reply:
Update tests and will add one comment to show fixing errors.